src/helpers/utils.ts

```diff
@@ -134,3 +134,13 @@ export function n(number, format = '(0.[00]a)') {
 export function ms(number) {
   return format(number * 1e3);
 }
+
+export function calcFromSeconds(value, unit) {
+  if (unit === 'h') return Math.floor(value / (60 * 60));
```

Comment:
I was wondering if we could use moment or date-fns to handle these.

Reply:
Can you please provide an example how that would be possible?